L1Trigger/TrackFindingTracklet/src/TrackletLUT.cc

```diff
@@ -1114,7 +1114,7 @@ void TrackletLUT::initVMRTable(unsigned int layerdisk, VMRTableType type, int re
     }
   }
 
-  if (settings_.combined()) {
+  if (settings_.combined() && combined) {
```

Comment:
Why do we need to pass variable "combined" around in this code, if we already have settings_.combined(). And why do we need to AND the two together? What is the difference between them?

Reply:
This `combined` flag is used to essentially ignore `settings_.combined()` in special cases. In particular, before this PR, `settings_.combined()` was hard-coded to be false for the extended tracking:
https://github.com/cms-L1TK/cmssw/blob/aac45c253340f46cfab4a748335dfe495ce6af11/L1Trigger/TrackFindingTracklet/plugins/L1FPGATrackProducer.cc#L286-L289

So this flag is used to revert to the previous behavior in the case of the triplet seeds, even though `settings_.combined()` is now properly set to true. This temporary hack will be removed in a subsequent PR in the near future.

Reply:
Fine, but please add comment to explain this at L1117, since otherwise settings.combined() && combined looks very weird.

Reply:
Done.